setup/inject_SELinux_AVC_check/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+BEAKERLIB_SCRIPT=/usr/share/beakerlib/logging.sh
+
+rlJournalStart
+
+    rlPhaseStartTest
+        rlLogInfo "Injecting the test code into rlPhaseStartTest"
+        rlRun "cp $BEAKERLIB_SCRIPT $BEAKERLIB_SCRIPT.pre_injection_backup.$$" 0 "Making backup of $BEAKERLIB_SCRIPT"
+        #for catching AVC
+        rlRun "setenforce 0"
+        if grep -q "__INTERNAL_TIMESTAMP_AVC" $BEAKERLIB_SCRIPT; then
+            rlRun "echo 'It was already injected into' $BEAKERLIB_SCRIPT"
+        else
+            rlRun "sed -i '/rlPhaseStart()/a\    __INTERNAL_TIMESTAMP_AVC=\`LC_ALL=en_US.UTF-8 date \"+%x %T\"\`\n    export __INTERNAL_TIMESTAMP_AVC\n' $BEAKERLIB_SCRIPT"
+            rlRun "sed -i '/^rlPhaseEnd()/a\    echo\n    echo \":: AVC SELinux denials since :: \$__INTERNAL_TIMESTAMP_AVC:\"\n    ausearch -m AVC -ts \$__INTERNAL_TIMESTAMP_AVC --input-logs && rlFail \"SELinux AVC check found denials!\"' $BEAKERLIB_SCRIPT"
```

Comment:
I think we could be more explicit stating that we are searching for AVC within a test phase. E.g.
  Test phase SELinux AVC denials since...
and
 .. found SELinux AVC denials within a test phase.
WDYT?

Reply:
I think that's good idea, I'll change it.